x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/outlier_exploration/outlier_exploration.tsx

```diff
@@ -55,57 +69,182 @@ export const OutlierExploration: FC<ExplorationProps> = React.memo(({ jobId }) =
     jobConfig !== undefined ? getFeatureCount(jobConfig.dest.results_field, tableItems) : 1
   );
 
+  const [expandedRowItem, setExpandedRowItem] = useState<DataFrameAnalyticsListRow | undefined>(
+    undefined
```

Comment:
No need to pass undefined explicitly here, I don't think.

Reply:
Fixed in b228018.